modules/standard/Set.chpl

```diff
@@ -346,7 +346,7 @@ module Set {
       :return: Whether or not this set and `other` intersect.
       :rtype: `bool`
     */
-    deprecated "Set isIntersecting() method is deprecated; use the negation of :proc:`isDisjoint` instead"
+    deprecated "Set isIntersecting() method is deprecated; use the negation of !:proc:`isDisjoint` instead"
```

Comment:
If the leading `!` works out, it doesn't need to say "the negation of" any more.